biz.aQute.bnd/test/aQute/bnd/junit/JUnitFrameworkTest.java

```diff
@@ -0,0 +1,150 @@
+package aQute.bnd.junit;
```

Comment:
Yeah, that sucks a bit but I do not see another way. I need to have all the repositories on the classpath + the embedded-repo. This creates a cycle. Although the code does not depend on it, our build depends on it. 

I would be nice if we had a single project that created the uber bnd jar that was then shared with bnd and gradle, and maybe bndtools.